runelite-client/src/main/java/net/runelite/client/plugins/idlenotifier/IdleNotifierPlugin.java

```diff
@@ -344,6 +349,15 @@ private void onPlayerSpawned(PlayerSpawned event)
 		}
 	}
 
+	private void onWallObjectSpawned(WallObjectSpawned event)
+	{
+		WallObject wall = event.getWallObject();
+		if (this.notifyResourceDoor && wall.getId() == 83 && resourceDoorReady)
```

Comment:
add in a Region check for this.

Reply:
region id is 12605

Reply:
I'll add this now, thanks

Reply:
Check fight caves as that implements a region check